test/gtest/ucp/test_ucp_sockaddr.cc

```diff
@@ -2841,6 +2841,13 @@ UCS_TEST_P(test_ucp_sockaddr_protocols_err, tag_rndv_unexp_put_scheme,
 }
 
 UCP_INSTANTIATE_CM_TEST_CASE(test_ucp_sockaddr_protocols_err)
+UCP_INSTANTIATE_TEST_CASE_TLS_GPU_AWARE(test_ucp_sockaddr_protocols_err,
+                                        dc_mlx5, "dc_mlx5,ud")
+UCP_INSTANTIATE_TEST_CASE_TLS_GPU_AWARE(test_ucp_sockaddr_protocols_err,
+                                        rc_mlx5, "rc_mlx5")
+UCP_INSTANTIATE_TEST_CASE_TLS_GPU_AWARE(test_ucp_sockaddr_protocols_err,
+                                        rc_verbs, "rc_verbs")
```

Comment:
do we realy need to add all of these? it increases testing time

Reply:
IMO, we need to test RC withoud UD keepalive.
combined `rc_mlx5` and `rc_verbs` to `rc_no_ud`

Reply:
ok for rc. but why do we need dc_mlx5,ud? isn't it covered by dc_x ?